### DISCUSSION TITLE
window启动器隧道创建成功，远程桌面无法链接

### DISCUSSION THREAD

**Discussion by q2521050661** [natfrp/wiki#506] [使用问题]
telnet端口显示通的，本地没有显示3389防火墙端口
![WX20240626-111131](https://github.com/natfrp/wiki/assets/31497741/7771863d-f1e6-4ce6-b705-57cfcdeed9e9)
![WX20240626-111238](https://github.com/natfrp/wiki/assets/31497741/796eb0ed-0b86-4934-96da-f3164b8cb8ce)

[a comment by an automated account is not shown]

**Comment by tianlongJ** [comment]
关闭自动https
